controllers/chat/initChatFromProfileAsAnonymousV2.js

```diff
@@ -46,15 +46,14 @@ const initChatFromProfileAsAnonymousV2 = async (req, res) => {
     const channel = client.channel('messaging', {members});
     const createdChannel = await channel.create();
 
-    await channel.updatePartial({
-      set: {
+    const {betterChannelMember, betterChannelMemberObject} =
+      await BetterSocialCore.chat.updateBetterChannelMembers(channel, createdChannel, true, {
         channel_type: CHANNEL_TYPE.ANONYMOUS,
         anon_user_info_color_code,
         anon_user_info_color_name,
         anon_user_info_emoji_code,
         anon_user_info_emoji_name
-      }
-    });
+      });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new implementation of `updateBetterChannelMembers` seems to be more modular and maintainable than the previous `channel.updatePartial` call. However, it's important to ensure that error handling is properly implemented in the `updateBetterChannelMembers` function. If an error occurs during the update process, it should be caught and handled appropriately to prevent the application from crashing or entering an inconsistent state.
```javascript
try {
  const {betterChannelMember, betterChannelMemberObject} =
    await BetterSocialCore.chat.updateBetterChannelMembers(channel, createdChannel, true, {
      channel_type: CHANNEL_TYPE.ANONYMOUS,
      anon_user_info_color_code,
      anon_user_info_color_name,
      anon_user_info_emoji_code,
      anon_user_info_emoji_name
    });
} catch (error) {
  // Handle error appropriately
}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->